pyproject.toml

```
@@ -14,6 +14,9 @@ prometheus-client = "^0.12.0"
 uvicorn = {extras = ["standard"], version = "^0.15.0"}
 python-multipart = "^0.0.5"
 PyYAML = "^6.0"
+mkdocs = "^1.4.2"
+mkdocs-material = "^9.1.6"
+mkdocs-multirepo-plugin = "^0.6.1"
```

As mentioned, they should be part of [a group](https://python-poetry.org/docs/managing-dependencies/)
```suggestion
[tool.poetry.group.docs]
optional = true

[tool.poetry.group.docs]
mkdocs = "^1.4.2"
mkdocs-material = "^9.1.6"
mkdocs-multirepo-plugin = "^0.6.1"
```

Then you can run
```shell
poetry install --with docs
```

The why, is because we do not want those dependencies to be installed at runtime. only to build the doc.